source/apiVolontaria/location/serializers.py

```diff
@@ -0,0 +1,92 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class CountryBasicSerializer(serializers.ModelSerializer):
+
+    """
+
+    This class represents the Country model serializer.
+
+    A serializer will format the object prior to the interaction with the user.
+
+    """
+
+    class Meta:
+        model = models.Country
+        fields = (
+            'iso_code',
+            'name',
+        )
+
+
+class StateProvinceBasicSerializer(serializers.ModelSerializer):
+
+    """
+
+    This class represents the StateProvince model serializer.
+
+    A serializer will format the object prior to the interaction with the user.
+
+    """
+
+    def to_representation(self, instance):
+        data = dict()
+        data['country'] = CountryBasicSerializer(
+            instance.country
+        ).to_representation(instance.country)
+        return dict(
+            iso_code=instance.iso_code,
+            name=instance.name,
+            **data
+        )
+
+    class Meta:
+        model = models.StateProvince
+        fields = (
+            'iso_code',
+            'name',
+            'country',
+        )
+
+
+class AddressBasicSerializer(serializers.ModelSerializer):
+
+    """
+
+    This class represents the Address model serializer.
+
+    A serializer will format the object prior to the interaction with the user.
```

Comment:
This line is not really usefull for developers, It's the same for all Serializers